frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -34,15 +34,17 @@ for (const mode of modeList) {
 export default class FeedbackCodeView extends Vue {
   @Prop() language!: string;
   @Prop() value!: string;
-  @Prop({ default: () => new Map<number, FeedbackClass>() }) feedbackMap!: Map<
-    number,
-    ArenaCourseFeedback
-  >;
+  @Prop({ default: () => new Map<number, typeof FeedbackClass>() })
+  feedbackMap!: Map<number, Feedback>;
   @Ref('cm-editor') private readonly cmEditor!: HTMLTextAreaElement;
 
   T = T;
   mode = languageModeMap[this.language] ?? languageModeMap['cpp17-gcc'];
-  numberOfComments = 0;
+  mapChangeTracker = 0;
+
+  get mapAsList(): 0 | [number, Feedback][] {
```

Comment:
```suggestion
  get feedbackList(): [number, Feedback][] {
```

si inicializamos `mapChangeTracker` a 1, podemos quitar el `0 |`? si no, podemos cambiar el código a que sea

```typescript
  if (!this.mapChangeTracker) {
    throw new Error('unreachable code');
  }
  return Array.from(this.feedbackMap);
```

Reply:
Me sigue marcando el mismo error. Así que mejor aplicamos la solución con el `throw`